app/src/main/java/com/zulip/android/networking/GravatarAsyncFetchTask.java

```diff
@@ -47,7 +48,7 @@ public static URL sizedURL(Context context, String url, float dpSize) {
         float px = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP,
                 dpSize, r.getDisplayMetrics());
         try {
-            return new URL(url + "&s=" + px);
+            return new URL(UrlHelper.addHost(url)  + "&s=" + px);
```

Comment:
yes we are checking whether url starts with http or not in the `addHost` method.
https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/util/UrlHelper.java#L8

Do we want any other checks?